### PR TITLE
chore(models): rewrite code format.

### PR DESCRIPTION
**Pull request by appleboy**
1. `var a` => `a:`
2. `//xxxx` => `// xxx`

### CODE REVIEW

**Comment by zeripath** [comment]
Ironically for a linting pr this one has caused a linting failure...

Following this pr are you able to remove any special exemptions for models/ from golangcilint?

**Comment by 6543** [comment]
>     1. `var a` => `a:`
> 
>     2. `//xxxx` => `// xxx`

`2.` I can understand ...

but what reason/impact do `1.` have?

@appleboy

**Comment by zeripath** [comment]
it's more a stylistic issue. In general in our code we use `a := ` instead of `var a =`

**Comment by 6543** [comment]
@appleboy pull got conflicts & CI still failing :/

**Comment by appleboy** [comment]
@6543 I will take it.

**Comment by appleboy** [comment]
@6543 Done. Please help to review.

**Comment by lunny** [comment]
> 1. `var a` => `a:`
> 2. `//xxxx` => `// xxx`

I like `var a = ` than `a :=` better :)

File: models/migrations/v158.go [hunk 1]
```diff
@@ -52,8 +52,8 @@ func updateCodeCommentReplies(x *xorm.Engine) error {
 		AND comment.commit_sha != first.commit_sha`
 
 	var sqlCmd string
```
Comment (lunny) [inline models/migrations/v158.go]: I would like 
```go
var (
  sqlCmd string
  start int
  batchSize = 100
  sess = x.NewSession()
)
```

Comment (appleboy) [reply models/migrations/v158.go]: See 22827c56a

**Review by lunny — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**